front/app/api/idea_statuses/useIdeaStatuses.ts

```diff
@@ -4,15 +4,29 @@ import { CLErrors } from 'typings';
 import fetcher from 'utils/cl-react-query/fetcher';
 
 import ideaStatusesKeys from './keys';
-import { IIdeaStatuses, IdeaStatusesKeys } from './types';
+import {
+  IIdeaStatuses,
+  IdeaStatusesKeys,
+  IdeaStatusesQueryParams,
+} from './types';
 
-const fetchIdeaStatuses = () =>
-  fetcher<IIdeaStatuses>({ path: '/idea_statuses', action: 'get' });
+const fetchIdeaStatuses = ({ participation_method }: IdeaStatusesQueryParams) =>
+  fetcher<IIdeaStatuses>({
+    path: '/idea_statuses',
+    action: 'get',
+    queryParams: { participation_method },
+  });
 
-const useIdeaStatuses = ({ enabled = true } = {}) => {
+const useIdeaStatuses = ({
+  enabled = true,
```

Comment:
I this a parameter that is sent in the API request? Because the backend does not support this. If so, then I think it can be removed?

Reply:
@sebastienhoorens It's [something for React Query actually](https://tanstack.com/query/v4/docs/framework/react/guides/disabling-queries) - it's used to enable or disable the request/query (on the FE). Given that we're always setting it to "true" though and I don't see us setting `enabled = false` anywhere where useIdeaStatuses is used, the request will always be made :thinking: 

@IvaKop Is this something we can remove, or alternatively, should we be setting this "enabled" value to something else? :)